ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -1,183 +1,174 @@
-{%             endif %}
-{%         endif %}
-{%         if port_channel_interfaces[port_channel_interface].shutdown is defined and port_channel_interfaces[port_channel_interface].shutdown == true %}
+{%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].shutdown is arista.avd.defined(true) %}
    shutdown
-{%         elif port_channel_interfaces[port_channel_interface].shutdown is defined and port_channel_interfaces[port_channel_interface].shutdown == false %}
+{%     elif port_channel_interfaces[port_channel_interface].shutdown is arista.avd.defined(false) %}
    no shutdown
-{%         endif %}
-{%         if port_channel_interfaces[port_channel_interface].mtu is defined and port_channel_interfaces[port_channel_interface].mtu is not none %}
+{%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].mtu is arista.avd.defined %}
    mtu {{ port_channel_interfaces[port_channel_interface].mtu }}
-{%         endif %}
-{%         if port_channel_interfaces[port_channel_interface].type is defined and port_channel_interfaces[port_channel_interface].type == "routed" %}
+{%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].type is arista.avd.defined("routed") %}
    no switchport
-{%         else %}
+{%     else %}
    switchport
-{%         endif %}
-{%         if port_channel_interfaces[port_channel_interface].mode is defined and port_channel_interfaces[port_channel_interface].mode == "access" %}
+{%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].mode is arista.avd.defined("access") %}
    switchport access vlan {{ port_channel_interfaces[port_channel_interface].vlans }}
-{%         endif %}
-{%         if port_channel_interfaces[port_channel_interface].vlans is defined and port_channel_interfaces[port_channel_interface].mode == "trunk" %}
+{%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].vlans is arista.avd.defined and port_channel_interfaces[port_channel_interface].mode is arista.avd.defined("trunk") %}
    switchport trunk allowed vlan {{ port_channel_interfaces[port_channel_interface].vlans }}
-{%         endif %}
-{%         if port_channel_interfaces[port_channel_interface].native_vlan is defined and port_channel_interfaces[port_channel_interface].mode == "trunk" %}
+{%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].native_vlan is arista.avd.defined and port_channel_interfaces[port_channel_interface].mode is arista.avd.defined("trunk") %}
    switchport trunk native vlan {{ port_channel_interfaces[port_channel_interface].native_vlan }}
-{%         endif %}
-{%         if port_channel_interfaces[port_channel_interface].mode is defined and port_channel_interfaces[port_channel_interface].mode == "trunk" %}
+{%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].mode is arista.avd.defined and port_channel_interfaces[port_channel_interface].mode is arista.avd.defined("trunk") %}
```

Comment:
Can be reduced to only `port_channel_interfaces[port_channel_interface].mode is arista.avd.defined("trunk")`

Reply:
fixed with commit c97751c